src/Package/Impl/DataInspect/VariableView.xaml

```diff
@@ -88,55 +89,48 @@
                     <TextBlock Grid.Column="2" Text="{Binding Model.Content.Name}" VerticalAlignment="Center" />
                 </Grid>
             </DataTemplate>
-
+            
             <DataTemplate x:Key="ValueCellTemplate">
                 <Grid Height="{Binding Path=RowHeight, ElementName=RootTreeGrid, Converter={x:Static rwpf:Converters.Scale155}}"
                       VerticalAlignment="Center"  ToolTip="{Binding Model.Content.Value}">
                     <Grid.ColumnDefinitions>
                         <ColumnDefinition Width="*" />
                         <ColumnDefinition Width="Auto" />
                     </Grid.ColumnDefinitions>
-                    <TextBlock Text="{Binding Model.Content.Value}" VerticalAlignment="Center" HorizontalAlignment="Stretch" Grid.Column="0" Padding="2,0,0,0"/>
-                    <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Grid.Column="1">
-                        <Button
+                    <TextBlock Text="{Binding Model.Content.Value}" VerticalAlignment="Center" HorizontalAlignment="Stretch" Grid.Column="0" Padding="2,0,2,0"/>
+                    <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Margin="2,0,0,0" Grid.Column="1">
+                        <StackPanel.Resources>
+                            <Style TargetType="{x:Type Button}">
+                                <Setter Property="Background" Value="Transparent"/>
+                                <Setter Property="BorderThickness" Value="1"/>
+                                <Setter Property="BorderBrush" Value="Transparent"/>
+                                <Style.Triggers>
+                                    <Trigger Property="IsMouseOver" Value="true">
+                                        <Setter Property="BorderBrush"
+                                Value="{DynamicResource {x:Static vsui:EnvironmentColors.ToolWindowButtonHoverActiveBorderBrushKey}}" />
+                                        <Setter Property="Background"
+                                Value="{DynamicResource {x:Static vsui:EnvironmentColors.ToolWindowButtonHoverActiveBrushKey}}" />
+                                    </Trigger>
+                                </Style.Triggers>
+                            </Style>
+                        </StackPanel.Resources>
+                        <StackPanel.Visibility>
+                            <MultiBinding Converter="{wpf:Lambda '(x, y) => x || y ? Visibility.Visible : Visibility.Collapsed'}">
```

Comment:
`{x:Static wpf:Converters.AnyIsNotCollapsed}`
Unfortunately, lambdas can't run in designer, so if we have converter for a common case, I'd prefer to use it instead of lambda